gateway/src/apicast/policy/keycloak_scope_check/apicast-policy.json

```diff
@@ -0,0 +1,63 @@
+{
+  "$schema": "http://apicast.io/policy-v1/schema#manifest#",
+  "name": "Keycloak Scope Check",
+  "summary": "Adds scope check with Keycloak.",
+  "description": [
+    "This policy adds scope check with Keycloak.\n",
+    "This policy exposes realm roles and client roles in the access token."
```

Comment:
This policy does not expose anything, but verifies those roles, right?

Reply:
Changed.